### ISSUE TITLE
No intellisense for module options in nuxt.config

### ISSUE THREAD

**Issue by dsvgl** [cpsoinos/nuxt-svgo#264]
Inside `nuxt.config.ts`, when I start typing `sv` my IDE (VS Code) correctly suggest `svgo?`. But inside this config object I then get no intellisense.

Maybe it's because of the way you typed the nuxt module in https://github.com/cpsoinos/nuxt-svgo/blob/main/src/module.ts#L55 ?

```ts 
const nuxtSvgo: NuxtModule<ModuleOptions> = defineNuxtModule({...});
```

I think it should rather be

```ts 
export default defineNuxtModule<ModuleOptions>({...});
```

**Comment by jd1378** [comment]
Should be fixed in v4.0.1

**Comment by dsvgl** [comment]
thx!
